build.sbt

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-val iotHubKafkaConnectVersion = "0.7.0"
+val iotHubKafkaConnectVersion = util.Properties.propOrNone("version").getOrElse("0.7.1")
```

Comment:
what does `util.Properties.propOrNone("version")` do? e.g. where is the value retrieved from?

Reply:
`util.Properties.propOrNone("version")` allows us to read params passed to the `sbt tool`. e.g. `sbt -Dversion=0.7.1`. Travis could pass this param in the `script` section when it calls sbt (currently it only passes scalaVersion).